e2e/commands/encounter-operations.ts

```diff
@@ -0,0 +1,60 @@
+import { type APIRequestContext, expect } from '@playwright/test';
+import { type Encounter } from './types';
+import { type Visit } from '@openmrs/esm-framework';
+import dayjs from 'dayjs';
+
+export interface Observation {
+  uuid: string;
+  concept: {
+    uuid: string;
+    display: string;
+    conceptClass: {
+      uuid: string;
+      display: string;
+    };
+  };
+  display: string;
+  groupMembers: null | Array<{
+    uuid: string;
+    concept: {
+      uuid: string;
+      display: string;
+    };
+    value: {
+      uuid: string;
+      display: string;
+    };
+  }>;
+  value: any;
+  obsDatetime: string;
+}
+
+export const createEncounter = async (
+  api: APIRequestContext,
+  patientId: string,
+  providerId: string,
+  visit: Visit,
+): Promise<Encounter> => {
+  const encounterRes = await api.post('encounter', {
+    data: {
+      encounterDatetime: dayjs().format(),
+      form: 'c75f120a-04ec-11e3-8780-2b40bef9a44b',
+      patient: patientId,
+      visit: visit,
```

Comment:
I think we can just use the visitUuid, but we should make sure that the encounterDatetime is slightly > the visit date time.

Reply:
thanks @ibacher ,, updated it